dragonfly/block/wood/wood.go

```diff
@@ -6,41 +6,52 @@ import "fmt"
 // others carry one of these types.
 type Wood struct {
 	wood
+	Flammable bool
```

Comment:
My only concern with bools like this is that is that the default `Wood` will not be flammable. This could cause some nasty bugs. Might be better to just have a method that returns whether it's flammable based on `wood`.

Reply:
The same counts for the other types where it's like this, but that's for later.